src/engine/runner/bootstrapper/index.ts

```diff
@@ -0,0 +1,11 @@
+import { ExpressBootstrapper } from "./express/bootstrapper";
+import { Configuration, ProjectType } from "../../configuration/index";
+
+export const getBootstraper = (configuration: Configuration) => {
+  switch (configuration.project) {
```

Comment:
I like to replace swithces like those with an object mapping. Personally I prefer this appraoch since you are not able to produce switch statement anti pattern and it is pretty short implementation.
```ts
const projects = {
   [ProjectType.Express]: ExpressBootstrap
};
const getBootstrap = (configuration) => new projects[configuration.project](configuration)
```

Reply:
But you lose the ability to throw an error when you don't match the key from the object. So you need to add additional logic to handle it. Let's leave it as it is now because it will change nothing in this case.

Reply:
No at all, I dont know why would you think that.
There is nothing that prevent you from throwing the error.
```ts
const projects = {
   [ProjectType.Express]: ExpressBootstrap
};
const getBootstrap = (configuration) => {
const constructor  = projects[configuration.project];
if(constructor === undefined) {
throw Error();
}
return new constructor(configuration);
}
```

Reply:
So we are reinventing a wheel. We can just use a built-in switch instead of writing a custom solution, that will be understood by all.